### PR TITLE
Fix spec

### PR DESCRIPTION
**Pull request by bpradipt**
Fix the PeerPodConfig spec. Also cleanup the deploy folder.

### CODE REVIEW

**Review by littlejawa — APPROVED** [review]
lgtm
Thanks @bpradipt

File: controllers/openshift_controller.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+
```
Comment (jensfr) [inline controllers/openshift_controller.go:22]: unintended?

**Review by jensfr — APPROVED** [review]
lgtm, thanks @bpradipt

File: controllers/openshift_controller.go (hunk 1 above)
Comment (bpradipt) [reply controllers/openshift_controller.go:22]: Yeah via vim-go formatting.

**Pull request merged**